packages/edge-login-ui-rn/src/reducers/CreateUserReducer.js

```diff
@@ -15,7 +15,8 @@ export type CreateState = {|
   +pinError: string,
   +pinErrorMessage: string | null,
   +username: string | null,
-  +usernameErrorMessage: string | null
+  +usernameErrorMessage: string | null,
+  +checkUsernameAvailabiltyPending: boolean
```

Comment:
This should be local scene state. We are trying to reduce the amount of stuff in Redux.